### PR TITLE
fix(google-photos): assorted changes

### LINKED ISSUE #906: Google Photos incorrect and missing styles
### Is there an existing issue outlining your problem?

- [X] I am using the latest version of the provided userstyle.
- [X] I have searched the existing issues and they do not solve my problem.

### What userstyles are you seeing the problem on?

lbl:google-photos

### Describe your problem.

The following pages and sections have un-styled background, wrong font color etc. Please check the attached screenshots.

* [Sharing page album title](https://photos.google.com/sharing) (including sharing a media)
* Albums: Title & Activity (sidebar)
* [Utilities: Manage library](https://photos.google.com/managelibrary)
  * [Highlight video](https://photos.google.com/movies/create)
  * Animation
  * Collage
  * Move photos to archive
  * [Back up photos from your computer](https://photos.google.com/apps)
* [Locked Folder (Move to Lock Folder)](https://photos.google.com/lockedfolder)
* [Trash](https://photos.google.com/trash)

### Attach screenshots.
sharing
sharing a media
albums
album
create video
create collage
create animation
Move photos to archive
Backup import photos
locked folder
trash
trash-default

### What browser(s) are you seeing the problem on?

Microsoft Edge v125.0.2535.51

### Any additional comments?

_No response_

### PR DESCRIPTION
**Pull request by GenShibe**
## 🔧 What does this fix? 🔧
closes #906 (i'm 99% sure anyways...)

## 🗒 Checklist 🗒

- [x] I have read and followed Catppuccin's [contributing guidelines](https://github.com/catppuccin/userstyles/blob/main/docs/CONTRIBUTING.md).
- [x] I have updated the version appropriately in the `==UserStyle==` header of the `catppuccin.user.css` file.

### CODE REVIEW

**Review by isabelroses — DISMISSED** [review]

**Review by uncenter — CHANGES_REQUESTED** [review]
There are a lot of classes here and almost all of them without comments. This is really quite unmaintainable. At this rate this userstyle will need a rewrite at some point soon purely do to us having to add on new classes and not knowing if it is safe to remove others...

**Comment by GenShibe** [comment]
> There are a lot of classes here and almost all of them without comments. This is really quite unmaintainable. At this rate this userstyle will need a rewrite at some point soon purely do to us having to add on new classes and not knowing if it is safe to remove others...

hopefully i fixed this with the new commit, feel free to nitpick as needed

**Review by isabelroses — APPROVED** [review]

File: styles/google-photos/catppuccin.user.css [hunk 1]
```diff
@@ -70,156 +70,487 @@
...
-      border-top-color: @surface0;
+    /* cannot connect page */
+    .Ih0h6e,
+    .aPRVsf {
+      color: @text;
+    }
+    /* rety button */
+    .RSxr8 {
+      background-color: darken(@blue, 15%);
     }
```
Comment (uncenter) [inline styles/google-photos/catppuccin.user.css:312]: Blue or accent? Typically blue is used as Google's accent color.

File: styles/google-photos/catppuccin.user.css [hunk 2]
```diff
@@ -70,156 +70,487 @@
...
+    .rN6Qmb,
+    .eT2rxc,
+    .EqBDpe,
+    .cuFkEd {
+      color: @subtext0;
+    }
+    /* "view" button links */
+    .oFJeZc:not(:disabled) {
+      color: @blue;
+    }
```
Comment (uncenter) [inline styles/google-photos/catppuccin.user.css:403]: Same as above, probably should be accent.

File: styles/google-photos/catppuccin.user.css [hunk 3]
```diff
@@ -70,156 +70,487 @@
...
+    /* create highlight video subpage */
+    .a4dmPd {
+      border-color: @surface0;
+    }
+    .BH0KJc {
+      color: @text;
+    }
+    .z3RRjd:disabled {
+      color: fade(@text, 25%);
+    }
```
Comment (uncenter) [inline styles/google-photos/catppuccin.user.css:519]: Just use `subtext0` here I think.

**Review by uncenter — CHANGES_REQUESTED** [review]

**Review by uncenter — CHANGES_REQUESTED** [review]
Again on the blue vs accent color thing - I'd just search for `@blue` in this entire userstyle and make sure any instances of it are really only for links or in SVGs. Most cases I can see scrolling through should be accent instead of blue.

**Review by uncenter — APPROVED** [review]

**Pull request merged**